packages/react-liveness/src/components/FaceLivenessDetector/service/types/machine.ts

```diff
@@ -74,6 +77,7 @@ export interface HydratedLivenessContext {
   faceMatchStateBeforeStart: FaceMatchState;
   isFaceFarEnoughBeforeRecording: boolean;
   isRecordingStopped: boolean;
+  theme: WebTheme;
```

Comment:
Why do we need the theme here? Seems wrong to pass it via the state machine context, why can't we just import it where it is used?

Reply:
Think this was leftover testing code. will remove!